tests/test-utils.jsx

```diff
@@ -3,7 +3,7 @@ import { Provider } from 'react-redux'
 import { configureStore } from '@reduxjs/toolkit'
 import reducer from '~/redux/reducer'
 import { StyledEngineProvider, ThemeProvider } from '@mui/material/styles'
-import { render, waitFor } from '@testing-library/react'
+import { screen, render, waitFor, fireEvent, act } from '@testing-library/react'
```

Comment:
You can remove 'render' from import.

Reply:
I can't remove it because it fails. But I found `vi` import and I removed it instead